app/uk/gov/hmrc/thirdpartydeveloperfrontend/service/SupportService.scala

```diff
@@ -59,8 +59,9 @@ class SupportService @Inject() (
     (
       for {
         flow       <- ET.liftF(fetchSupportFlow(sessionId))
-        api        <- ET.fromEitherF(apmConnector.fetchExtendedApiDefinition(apiChoice))
-        updatedFlow = flow.copy(api = Some(SupportApi(apiChoice, api.name)))
+        apiName    <- if (apiChoice.value == "api-not-in-list") ET.liftF(Future.successful(""))
```

Comment:
ET.pure("")

Reply:
This doesn't work. There's an error about the _left_ type becoming a Serializable instead of a Throwable. Possibly a problem in the library? Or combination of ET.pure with if/else